website/content/partials/known-issues/autopilot-upgrade-upgrade-version.mdx

```diff
@@ -0,0 +1,16 @@
+### New nodes added by autopilot upgrades provisioned with the wrong version
+
+#### Affected versions
+
+- 1.15.3 - 1.15.8
+- 1.16.1 - 1.16.2
```

Comment:
what about 1.15.9 and 1.16.3?

Reply:
Depending on release of those versions, won't this be patched? Or will it go in the next release after?

Reply:
They did! updated.